BurnOutSharp/PackerType/WinZipSFX.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BurnOutSharp.PackerType
+{
+    public class WinZipSFX : IContentCheck, IScannable
+    {
+        /// <inheritdoc/>
+        public bool ShouldScan(byte[] magic) => true;
+
+        /// <inheritdoc/>
+        public string CheckContents(string file, byte[] fileContent, bool includePosition = false)
+        {
+            // Every single version has the "WinZip Self-Extractor", and every 3+ version has a "version=" string due to internally including an XML
+            // "WinZip Self-Extractor"
+            byte[] check = new byte[] { 0x57, 0x69, 0x6E, 0x5A, 0x69, 0x70, 0x20, 0x53, 0x65, 0x6C, 0x66, 0x2D, 0x45, 0x78, 0x74, 0x72, 0x61, 0x63, 0x74, 0x6F, 0x72 };
+            if (fileContent.Contains(check, out int position))
+            {
+                // "version="
```

Comment:
Because of the below version shennanigans, is the presence of `_winzip_` a more consistent indicator of <V2.x compared to >3.x? Or was that something entirely separate?

Reply:
As per convseration in Discord, would the presence of the XML header be a better extension to this check rather than just `version=`?

Reply:
Same here.